src/SFA.DAS.EAS.DbMaintenance.WebJob/IdProcessor/IBroker.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Threading.Tasks;
+
+namespace SFA.DAS.EAS.DbMaintenance.WebJob.IdProcessor
+{
+    /// <summary>
+    ///     Obtain account ids and for each invoke a specified processor.
```

Comment:
Remove "account" from "account ids" in summary

Reply:
Done